bot/exts/evergreen/githubinfo.py

```diff
@@ -74,24 +83,76 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
 
                 embed.add_field(name="Followers",
                                 value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)")
-                embed.add_field(name="\u200b", value="\u200b")
                 embed.add_field(name="Following",
                                 value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)")
 
             embed.add_field(name="Public repos",
                             value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)")
-            embed.add_field(name="\u200b", value="\u200b")
 
             if user_data['type'] == "User":
                 embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
 
                 embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
                                 value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
             embed.add_field(name="Website", value=blog)
 
         await ctx.send(embed=embed)
 
+    @github_group.command(name='repository', aliases=('repo',))
+    async def github_repo_info(self, ctx: commands.Context, repo: str) -> None:
+        """
+        Fetches a repositories's GitHub information. Repository should look like `user/reponame`.
+
+        Repository is optional and sends the help command if not specified.
+        """
+        async with ctx.typing():
+            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{repo}")
+
+            # There won't be a message key if this repo exists
+            if repo_data.get('message') is not None:
+                embed = discord.Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="The requested repository was not found.",
+                    colour=discord.Colour.red()
```

Comment:
Should do:

Use `constants.Colours.soft_red` instead, to maintain the similarity in Error Embeds.
https://github.com/python-discord/sir-lancebot/blob/a059eb2dc1caf2d8012629e311ac9b4a0b164692/bot/constants.py#L144